data/json/items/tool/deployable.json

```diff
@@ -331,5 +331,41 @@
     "symbol": "T",
     "color": "red",
     "flags": [ "DURABLE_MELEE" ]
+  },
+  {
+    "id": "beverly_shear",
+    "type": "TOOL",
+    "name": { "str": "Beverly shear" },
+    "description": "A massive bench-mounted device with a short guillotine blade mounted to a long lever.  Designed for making straight or curved cuts through sheet metal up to 3 millimeters thick.",
+    "//": "Based on the Beverly B-2 Throatless Shear, 10 gauge capacity - as was furniture - values based on real world Beverly B-2 Throatless Shear",
+    "weight": "15875 g",
+    "volume": "50 L",
```

Comment:
Volume refers to the actual material volume.
Looking at the size mentioned [here](https://www.trick-tools.com/Beverly_B_2_Throatless_Shear_10_gauge_capacity_B_2_1118), it's bounding box is ca. 10x10x40 inches ~= 25x25x100 cm but I would assume its actual volume is closer to 25x25x25 cm + 5x5x75 cm due to the handle.
That would come out to about 18l

Reply:
I was thinking in item "bulkiness" category, but if actual item volume matters (I saw discussion and link to item dimensioning guide on Discord) then it is a reasonable change. I adjusted the code and used value you suggested.